scripts/crawl_expand.py

```diff
@@ -37,20 +39,21 @@ def expandPlaces(config, center, radius_km):
     placeIDs = geo.get_place_ids_in_radius(center, radius_km, location_table)
 
     log.info("{} places found".format(len(placeIDs)))
-    for placeID in placeIDs:
+
+    def fetchDetails(placeID):
```

Comment:
(spoke IRL) Make sure `fetchDetails` is thread safe (e.g. firebase, TA, yelp APIs correctly handle concurrent reads/writes)